projects/diu-component-library/src/lib/_components/_layouts/div-row.component.ts

```diff
@@ -39,6 +39,7 @@ export class DivRowComponent implements OnInit {
             fetchChildren(this.Handler.config.children, this.apiService).then((result: any) => {
                 this.Handler.children = result;
                 this.changeDetection.detectChanges();
+                console.log(result,this.Handler.children);
```

Comment:
Sorry Luke, you may not have meant to keep this in beyond testing locally but can you please remove the console.log unless it's absolutely needed